model/src/main/kotlin/Package.kt

```diff
@@ -63,8 +63,8 @@ data class Package(
     val authors: SortedSet<String> = sortedSetOf(),
 
     /**
-     * The list of licenses the authors have declared for this package. This does not necessarily correspond to the
-     * licenses as detected by a scanner. Both need to be taken into account for any conclusions.
+     * The list of licenses declared for this package. This does not necessarily correspond to the licenses as detected
```

Comment:
I would change "declared for this package" to "declared in the metadata for this package" to make it clear what we mean with "declared" as in the industry declared can also mean the main license e.g. LICENSE, COPYING, etc. [SPDX Declared License 7.15.2](https://spdx.github.io/spdx-spec/v2.3/package-information/#715-declared-license-field): "This is simply the license identified in text in one or more files (for example COPYING file) in the source code package."

Reply:
Is this really necessary as the `Package`'s class docs already say to "contain[s] all relevant *metadata* about a package"? I mean, we'd need to add that to each and every field then. Finally, this is not end-user documentation, but developer documentation.

Reply:
@tsteenbe would you be fine with merging this as-is, because I have a few changes upcoming that build upon these docs?

Reply:
@sschuberth  Change this PR to approved.